src/Redux/Missions/missionsActions.js

```diff
@@ -0,0 +1,15 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const URL = 'https://api.spacexdata.com/v3/missions';
+
+const getMissions = createAsyncThunk('missions/getMissions', async () => {
+  try {
+    const response = await axios.get(URL);
+    return response.data;
+  } catch (error) {
+    return error;
+  }
+});
+
```

Comment:
- [ ] Kindly filter out the mission attributes based on [the requirement here](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#redux-fetch-data-and-update-redux-store). It is not all the data from the API that should be saved to the store.

  ![image](https://user-images.githubusercontent.com/54848363/231276892-77a81ec1-612d-467d-8791-b20c1f6c7173.png)